src/core/App.js

```diff
@@ -54,14 +54,12 @@ class App extends ModuleSystem {
         };
     })();
 
-    const {loops, enabled} = this;
-
     function process() {
       requestAnimFrame(process);
-      if (!enabled) return;
+      if (!this.enabled) return;
```

Comment:
@pbalmasov Travis CI fails because it doesn't work (`process` function has another context and `this` becomes an empty object). Try solving that with `.bind` or just something like `const self = this;`